src/guide/components/v-model.md

```diff
@@ -27,7 +27,7 @@ The parent can then bind a value with `v-model`:
 
 ```vue-html
 <!-- Parent.vue -->
-<Child v-model="count" />
+<Child v-model="model" />
```

Comment:
```suggestion
<Child v-model="countModel" />
```

Reply:
@EtherNoteWorld Can you update the PR to update the variables so that it reflects `countModel` rather than just `model`?